docs/settings.rst

```diff
@@ -78,6 +78,7 @@ Setting name (followed by default value, if any)
                                                                                  date and the slug, you could set something like:
                                                                                  ``'(?P<date>\d{4}-\d{2}-\d{2})_(?P<slug>.*)'``.
                                                                                  See :ref:`path_metadata` and ``SLUGIFY_SOURCE``.
+``FORMATTED_FIELDS = []``                                                        A list of metadata fields to be parsed and translated to HTML.
```

Comment:
This setting (with specified default value) should be specified in settings.py also.